### PR TITLE
first steps of bookings controller

### PR DESCRIPTION
**Pull request by PeteTaylorson**

### CODE REVIEW

File: app/controllers/booking_controller.rb [hunk 1]
```diff
@@ -1,2 +1,28 @@
...
+end
+
+private
+
+def set_venue
+  @venue = Venue.find(params[:id])
+end
+
+def venue_params
+  params.require(:venue).permit(:name, :date)
```
Comment (Sols777) [inline app/controllers/booking_controller.rb]: image
 shouldnt the params also contain the rest of the fields like address ,description , capacity and bustability.

**Review by Sols777 — CHANGES_REQUESTED** [review]
add  address ,description , capacity and bustability.
remove date

**Comment by PeteTaylorson** [comment]
Yeah, it wasn't finished - only the 'first steps'

Just wanted to push at the end of the day.

**Comment by PeteTaylorson** [comment]
But actually no - this is bookings Controller not venues. Only date and
name with user ref and venue ref.

**Comment by PeteTaylorson** [comment]
Yeah this is incorrect, sorry.

**Review by Sols777 — COMMENTED** [review]
looks good man

**Pull request merged**